src/__stories__/components/Colors.tsx

```diff
@@ -40,45 +40,42 @@ const Colors = () => {
                   >
                     <Typography
                       color={readableColor(
-                        (
-                          theme.colors as Record<string, Record<string, string>>
-                        )[sentiment][context],
+                        theme.colors[sentiment as Color][
+                          context as AvailableContexts
+                        ],
```

Comment:
```suggestion
                        theme.colors[sentiment][context],
```
with typing done at the source, see https://github.com/scaleway/scaleway-ui/pull/1280/files#r828183432